it seems there is issue from kubevirtci,

we can see below error in above jobs:
Errors during downloading metadata for repository 'isv_cri-o_stable_v1.32':
  - Status code: 404 for https://storage.googleapis.com/kubevirtci-crio-mirror/isv_cri-o_stable_v1.32/repodata/repomd.xml (IP: 142.250.26.207)
Error: Failed to download metadata for repo 'isv_cri-o_stable_v1.32': Cannot download repomd.xml: Cannot download repodata/repomd.xml: All mirrors were tried
make: *** [Makefile:221: cluster-up] Error 1

I don't see isv_cri-o_stable_v1.32 in https://storage.googleapis.com/kubevirtci-crio-mirror/

but i see isv_cri-o_stable_v1.34 does exits,

do we need to upgrade kubevirtci version from 1.32 to 1.34
https://github.com/nmstate/kubernetes-nmstate/blob/ab814478a71981ef107b296178fdb6f48794fead/cluster/kubevirtci.sh#L1

--- a/build/install-go.sh
+++ b/build/install-go.sh
@@ -1,21 +1,17 @@
 #!/bin/bash -xe
 
 version=$1
-arch=""
+arch=$(uname -m)
 os=linux
-go_mod_version=$1
 
 dnf install -y jq
 
-case $(uname -m) in
-    x86_64)  arch="amd64";;
+case $arch in
+    x86_64)  arch="amd64" ;;
     aarch64) arch="arm64" ;;
 esac
 
-if [ "$arch" == "" ]; then
-    echo "Unknown architecture $(uname -m)"
-    exit 1
-fi
+echo "Installing Go ${version} for architecture: ${arch}"
 
 tarball_url="https://go.dev/dl/go${version}.${os}-${arch}.tar.gz"
 

--- a/hack/lint.sh
+++ b/hack/lint.sh
@@ -1,11 +1,10 @@
 #!/bin/bash -xe
-architecture=""
-case $(uname -m) in
-    x86_64)           architecture="amd64" ;;
-    aarch64|arm64)    architecture="arm64" ;;
-    *)
-                      echo "Unsupported architecture: $(uname -m), must be (x86_64|aarch64|arm64)."
-                      exit 1;
+
+architecture=$(uname -m)
+
+case $architecture in
+    x86_64)        architecture="amd64" ;;
+    aarch64|arm64) architecture="arm64" ;;
 esac
 
 path_combined="$(uname -s | tr '[:upper:]' '[:lower:]')-${architecture}"

